src/gui/LobbyScreen.java

```diff
@@ -1,34 +1,78 @@
 package gui;
 
 import java.util.ArrayList;
-import java.util.Arrays;
```

Comment:
Why was this import removed? Looks like it's still needed for line #42 in new file.